domain/src/main/java/com/example/domain/usecase/CharacterUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.domain.usecase
+
+import com.example.domain.entity.CharacterDataWrapperEntity
+import com.example.domain.utils.Result
+
+interface CharacterUseCase {
```

Comment:
for the useCase you can put in the same file the interface and the implementation

Reply:
done